examples/zephyr/arm-x86-64-eabi-gcc.cmake

```diff
@@ -0,0 +1,106 @@
+#
```

Comment:
I don't understand the filename, can you elaborate? The toolchain tuple naming convention - https://wiki.gentoo.org/wiki/Embedded_Handbook/Tuples#:~:text=From%20Gentoo%20Wiki,userland%20and/or%20userland%20ABI.